packages/relay/tests/lib/eth/eth-helpers.ts

```diff
@@ -62,8 +99,9 @@ export function generateEthTestEnv(fixedFeeHistory = false) {
   const duration = constants.HBAR_RATE_LIMIT_DURATION;
   const total = constants.HBAR_RATE_LIMIT_TINYBAR;
   const hbarLimiter = new HbarLimit(logger.child({ name: 'hbar-rate-limit' }), Date.now(), total, duration, registry);
+  const eventEmitter = new EventEmitter();
 
-  const hapiServiceInstance = new HAPIService(logger, registry, hbarLimiter, cacheService);
+  const hapiServiceInstance = new HAPIService(logger, registry, hbarLimiter, cacheService, eventEmitter);
```

Comment:
Seems like an unrelated change, is this a bug you realized that you're fixing.
Is so please update the PR description with the details.

Reply:
Yes, `eth-helpers.ts` wasn't updated to pass the new `EventEmitter` parameter of `HAPIService` when those changes which modified the constructor were introduced, and I was getting TypeScript errors for this file.